sentry-sample/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import org.jetbrains.kotlin.config.KotlinCompilerVersion
 plugins {
     id("com.android.application")
     kotlin("android")
-//    id("io.sentry.android.gradle") how to add sentry gradle plugin
+    id(Config.PublishPlugins.bintrayPlugin)
```

Comment:
this should go into the modules that would be published, not in the sample app.

Reply:
right, just playing around. Draft PR

Reply:
I'll add a description of the goal